raccoglitesi.go

```diff
@@ -71,7 +71,8 @@ func getDipartimenti() []Dipartimento {
 			re := regexp.MustCompile(`http[s]:\/\/(.*?)\.unibo`)
 			match := re.FindStringSubmatch(linkURL)
 			if len(match) != 2 {
-				log.Fatal("La pagina dei dipartimenti è probabilmente cambiata, non posso proseguire")
+				fmt.Fprintln(os.Stderr, "La pagina dei dipartimenti è probabilmente cambiata, non posso proseguire")
```

Comment:
La congiunzione con virgola (sentence-splice) è un errore
in italiano. Magari usare i due punti anziché la virgola
(sì, so che era già così).